src/Payum/Action/Api/PayWithCardAction.php

```diff
@@ -8,38 +8,38 @@
 use CommerceWeavers\SyliusTpayPlugin\Payum\Request\Api\PayWithCard;
 use CommerceWeavers\SyliusTpayPlugin\Tpay\PayGroup;
 use Payum\Core\Reply\HttpRedirect;
+use Payum\Core\Request\Generic;
 use Sylius\Component\Core\Model\PaymentInterface;
 use Webmozart\Assert\Assert;
 
-class PayWithCardAction extends BaseApiAwareAction
+class PayWithCardAction extends BasePaymentAwareAction
```

Comment:
Final maybe?

Reply:
In theory it can be, but we can skip that in this PR